scripts/send-message-from-rialto-millau.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Another approach to take here is to expand the existing `send-message` script to take a network argument.

Usage would be something like: `./send-message.sh remark --to rialto`

Reply:
This would be nice, but can be done in a follow up